### PR TITLE
[services] make snmp.timer work again and delay telemetry.service

### PR DESCRIPTION
**Pull request by stepanblyschak**
**- What I did**
Delay CPU intensive services at boot

**- How I did it**
Made snmp.timer work and add telemetry.timer.
But this is not enough because it breaks the existing snmp dependency on swss.
So, in this solution snmp timer is a wanted by swss service, but since OnBootSec timer expires only once it will not trigger snmp service, so I added line "OnUnitActiveSec=0 sec" which will start snmp service based on the last time it was active. On boot only OnBootSec will expire, on swss start/restarts only second timer will expire immediately and trigger snmp service.
However, snmp service will not stop after "systemctl stop snmp" because of the second timer which will always expire when snmp service because unavailable.
So there is a conflict which will be handled by systemd if we add "Conflicts=" line to both snmp.service and snmp.timer.

So during boot:
- snmp does not start by default
- swss starts and starts snmp timer
- OnUnitActiveSec=0 does not expire since there is no snmp active
- OnBootSec expires and starts snmp service and snmp timer gets stopped

During "systemctl restart swss"
- snmp stops because of Requisite on swss
- snmp unblocks snmp timer from running
- swss starts and starts snmp timer
- OnUnitActiveSec=0 expires imidiately and start snmp which stops snmp timer

During "systemctl stop snmp"
- stop of snmp service unblocks snmp timer but no one starts the timer so it is not started by "OnUnitActiveSec=0"

**- How to verify it**

- systemctl stop snmp - stops snmp, swss is still running
- systemctl start snmp - starts snmp, swss is still running
- systemctl stop swss - snmp stops with swss
- systemctl start swss - snmp starts with swss
- systemctl restart swss - snmp stops before swss and start after swss
- bootup - snmp & telemetry starts with 3.5 +- 1min delay

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: files/build_templates/snmp.service.j2 [hunk 1]
```diff
@@ -12,7 +13,4 @@ ExecStartPre=/usr/bin/{{docker_container_name}}.sh start
 ExecStart=/usr/bin/{{docker_container_name}}.sh wait
 ExecStop=/usr/bin/{{docker_container_name}}.sh stop
 Restart=always
-RestartSec=30
-
-[Install]
-WantedBy=multi-user.target swss.service
+RestartSec=30
```
Comment (jleveque) [inline files/build_templates/snmp.service.j2]: Please add newline at EOF

Comment (stepanblyschak) [reply files/build_templates/snmp.service.j2]: Done

File: files/build_templates/snmp.service.j2 [hunk 2]
```diff
@@ -14,5 +15,3 @@ ExecStop=/usr/bin/{{docker_container_name}}.sh stop
 Restart=always
 RestartSec=30
 
-[Install]
-WantedBy=multi-user.target swss.service
```
Comment (yxieca) [inline files/build_templates/snmp.service.j2]: By remove these 2 lines, restarting swss will no longer restarting snmp. You need to add snmp as a dependent of swss similar https://github.com/Azure/sonic-buildimage/blob/master/files/scripts/swss.sh#L5. But you need to add a logic to ignore this dependency if the uptime is less than 30 seconds.

Comment (stepanblyschak) [reply files/build_templates/snmp.service.j2]: If we will ignore snmp dependency in swss.sh if uptime is less than 210 sec, then who will start snmp? looks like we still need snmp.timer so the complexity will not decrease and we will have 210 sec hard coded in two places.
If the desired behavior can be done with systemd then why to do that in scripts?

**Pull request merged**